### DISCUSSION TITLE
Workflow [Jul 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2994] [Discord Threads]
> 🙋‍♂️ **Elf**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@160755405033701377> NocoDB supports webhook's. You can configure webhook's to be triggered on insert/delete/update to a record in a table.    
####    
#### You might want to write a handler using NocoDB SDK to trigger updates to Table-2 after receiving webhook (outside NocoDB server that is).  
#  
> 🙋‍♂️ **perlfan**  
#### Isn't there a on click feature? if someone clicks a button to add a record, you could execute your code.  
#  
> 👨‍🦲 **dstala**  
#### <@1001083667466174494> custom source code hooks are currently not supported.  
#  
> 🙋‍♂️ **Elf**  
#### Great! Thx  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1001359042234568765)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by lcfd** [comment]
> NocoDB supports webhook's. You can configure webhook's to be triggered on insert/delete/update to a record in a table.

So you suggesting is to trigger webhooks that point to NocoDB itself?
